src/components/Avatar/Avatar.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen } from "@testing-library/react"
+import { Avatar, AvatarSize } from "./Avatar"
+import { getInitials } from "./utils/getInitials"
+
+describe("Avatar", () => {
+  it("renders an avatar with initials", () => {
+    // Arrange
+    const name = "Crema Components"
+    const initials = getInitials(name)
+
+    // Act
+    render(<Avatar name={name} />)
+
+    const received = screen.getByText(initials)
+
+    // Assert
+    expect(received).toBeInTheDocument()
+  })
+  it("does not render initials if image is supplied", () => {
+    // Arrange
+    const name = "Crema Components"
+    const initials = getInitials(name)
+
+    // Act
+    render(<Avatar name={name} src="someUrl" />)
+
+    const received = screen.queryByText(initials)
+
+    // Assert
+    expect(received).not.toBeInTheDocument()
+  })
```

Comment:
💭 Completely subjective stylistic observation and fuel for conversation...

There is more whitespace within your test blocks than between them. This makes it hard for me to scan this file and discriminate between tests. But this may just be me and my subjective expectations. 🤷